nes_bnf.html

```diff
@@ -0,0 +1,1206 @@
+        rawtags.forEach( function(tag) {
+            var all = document.getElementsByTagName(tag);
+            for (var i=0, max=all.length; i < max; i++)
+                all[i].setAttribute('style','display:'+rawvalue);
+        })
+    }
+
+    function turnAccordinglyXML() {
+        var elem = document.getElementById('ShowXMLOnOff');
+        turnOnOffXML(elem.checked);
+    }
+
+    function turnOnOffHelp(on_off) {
+        // true - pokaz
+        // false - ukryj
+        // undefiined - trigger
+        var helpwnd = document.getElementById('HelpWindow');
+        if ( on_off === undefined )
+            on_off = helpwnd.style.display == 'none';
+        var tagvalue = on_off ? '' : 'none';
+        helpwnd.setAttribute('style','display:'+tagvalue);
+    }
+
+
+</script>
+
+
+</head>
+
+<body>
+
+
+<div class="all">
+<style>
+
+    raw  { font-family: Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace; }
+    ptag { font-family: Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace; }
+    ktag { font-family: Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace; }
+    pptag { font-family: Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace; }
+    pktag { font-family: Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace; }
+    ptag:before { content: "<"; }
+    ptag:after  { content: ">"; }
+    ktag:before { content: "</"; }
+    ktag:after  { content: ">"; }
+    pptag:before { content: "<"; }
+    pktag:before { content: ">"; }
+    attrib:before { content: "attribute."; }
+
+    uzycie { font-family: "Times New Roman", Georgia, Serif;
+             font-style: italic;
+             text-decoration: underline red;
+    }
+
+    definicja { font-family: "Times New Roman", Georgia, Serif;
+                font-style: italic;
+                text-decoration: underline red;
+    }
+
+    attrib { font-family: "Times New Roman", Georgia, Serif;
+             font-style: oblique;
+             font-weight: bold;
+    }
+
+    ptekst { font-size: 150%;
+    }
+
+    pteksti { font-size: 120%;
+    }
+
+    div.scrolledMain {
+        background-color: #ccFFFF;
+        height: 700px;
+        overflow: scroll;
+    }
+
+    div.scrolledDefaultHidden {
+        background-color: #bbf7ce;
+        height: 300px;
+        overflow: scroll;
+    }
+
+    div.przywolanaDef { background-color: #d1d1e0; }
+    div.uzywajaceDef  { background-color: #f1d1e0; }
+
+    .rozwinDev { background-color: #d1d1e0; }
+
+    .container  { width:90%; }
+    .leftButton { float: left;  background-color: #bbf7ce; }
+    .rightButton{ float: right; width: 250px; background-color: yellow; font-size: 140%; font-weight: bold; }
+
+    }	
+</style>
+
+<h1>Niezbędne elementy struktury dokumentów elektronicznych</h1>
+<h2><i>wg Rozporządzenia MSWiA z dnia 30 października 2006 r. (propozycja modyfikacji 28.III.2017, wersja 24.IV.2017)</i></h2>

```

Comment:
Tag `<i>` ma obecnie znaczenie semantyczne, a nie stylizujące (por. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) - jeśli chodzi tutaj o kursywę, to użycie klasy CSS byłoby lepsze.